docs/bigcommerce-for-wordpress/setup/multi-site.md

```diff
@@ -167,11 +167,10 @@ After clicking the text, you will be asked for your store's API credentials and
 
 ![](//s3.amazonaws.com/user-content.stoplight.io/6116/1544043952871 "")
 
-2. Enter the API credentials on your WordPress site. Saving the API credentials on your WordPress site will direct you to name the channel that the plugin will create. This allows you to list product to the channel from within BigCommerce and link orders back to the channel that come from the WordPress site. You can also link to an existing channel.
+2. Enter the API credentials on your WordPress site. Saving the API credentials on your WordPress site will direct you to name the channel that the plugin will create. This allows you to list product to the channel from within BigCommerce and link orders back to the channel that comes from the WordPress site. You can also link to an existing channel.
 
 _Congratulations, you're done setting up your additional site!_ 
 
-<div class="HubBlock--callout">
 <div class="CalloutBlock--">

```

Comment:
```suggestion
<div class="HubBlock--callout">
<div class="CalloutBlock--info">

```
restore 1st of 3 nested div tags that was accidentally deleted.  Add theme specifier to CalloutBlock class.